scripts/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Template build script
+# ---------------------
+
+rm -rf CMakeCache.txt CMakeFiles
+
+export CC=<FIXME: path to C compiler>
+export CXX=<FIXME: path to C++ compiler>
+export MKLROOT=<FIXME: path to the root installation directory of MKL>
+HPX_ROOT=<FIXME: path to the root installation directory of HPX>
+BLASPP_ROOT=<FIXME: path to the root installation directory of BLASPP>
+LAPACKPP_ROOT=<FIXME: path to the root installation directory of LAPACKPP>
+
+# CMAKE_BUILD_TYPE := Release | RelWithDebugInfo | Debug
+#
+# DLAF_WITH_CUDA   := ON | OFF (default)
+#
+# DLAF_WITH_MKL    := ON | OFF (default)
+#
+# BUILD_DOC        := ON | OFF (default)
+#
+# DLAF_WITH_TEST   := ON (default) | OFF 
+#
+cmake <FIXME: path to DLA-Future src directory> \
+  -D DLAF_WITH_MKL=ON \
+  -D CMAKE_BUILD_TYPE=RelWithDebugInfo \
+  -D CMAKE_PREFIX_PATH="$HPX_ROOT;$BLASPP_ROOT;$LAPACKPP_ROOT" \
```

Comment:
If you call the variables `_DIR` instead of `_ROOT`, it is not necessary to add them neither as CMake cache variable or in `CMAKE_PREFIX_PATH`.

In particular, variables names should be:
- `HPX_DIR`
- `blaspp_DIR`
- `lapackpp_DIR`